queries.sql

```diff
@@ -28,4 +28,49 @@ WHERE name <> 'Gabumon';
 
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/206004567-321513d3-c448-4f44-8462-8cbd10cb60e5.PNG)

- [ ] You didn't make these queries. Kindly follow these requirements 👍